src/main/webapp/go.js

```diff
@@ -48,10 +65,89 @@ function getHunt() {
       puzzleArr.push(mssg.items[i].riddle.puzzle);
       destArr.push(mssg.items[i].name);
     }
+    createMap();
     updateToCurrentState(destIndex);
   });
 }
 
+/**
+ * Add a marker to the map at the specified location.
+ * @param {Double} destLat: latitude of location.
+ * @param {Double} destLng: longitude of location.
+ * @param {String} destName: Name of location.
+ */
+function addMarkerToMap(destLat, destLng, destName) {
+  const coord = new google.maps.LatLng(destLat, destLng);
+  const marker = new google.maps.Marker({
+    position: coord,
+    title: destName,
+  });
+  marker.setMap(map);
+}
+
+/**
+ * Creates a map and adds it to the page.
+ */
+function createMap() {
+  map = new google.maps.Map(
+      document.getElementById(MAP_DISPLAY),
+      // Centered at GooglePlex (updated below).
+      {center: {lat: 37.422, lng: -122.084}, zoom: 7},
+  );
+  if (navigator.geolocation) {
+    navigator.geolocation.getCurrentPosition((position) => {
```

Comment:
I think it would be better to use `setMyLocationEnabled` so that the location is updated when the user moves. https://developers.google.com/android/reference/com/google/android/gms/maps/GoogleMap#public-void-setmylocationenabled-boolean-enabled

Reply:
I added a call to window.setInterval() that updates the user's location on the map every 10 seconds.